patterns/components/hero-carousel-small.php

```diff
@@ -0,0 +1,52 @@
+<?php if (have_rows('carousel_slides')): ?>
+  <section class="hero-carousel">
+    <div class="carousel rel">
+      <div class="carousel__slides js-carousel__single-item">
+        <?php
+          while (have_rows('carousel_slides')): the_row();
+            $image = get_sub_field('photo');
+            $title = get_sub_field('title');
+            $subtitle = get_sub_field('subtitle');
+            $description = get_sub_field('description');
+            $link_text = get_sub_field('link_text');
+            $link_url = get_sub_field('link_url');
+        ?>
+          <div class="carousel__item rel">
+            <picture class="picture">
+              <!--[if IE 9]><video style="display: none;"><![endif]-->
+              <source srcset="<?php echo $image['sizes']['featured__hero--xl']; ?>" media="(min-width: 1100px)">
```

Comment:
Don't think we'll need a `l` or `xl` option here if this remains a small gallery. Might not even need the `m` depending on how big that one is.